glass-easel/src/event.ts

```diff
@@ -239,77 +239,79 @@ export class Event<TDetail> {
     return this._$eventBubblingControl.mutated
   }
 
+  callListener(
```

Comment:
Private method? Add `private _$` if so.

Reply:
> Private method? Add `private _$` if so.

It's a public method for dual-thread backend usage.

Reply:
Why?

Reply:
> Why?

For a dual-thread backend, events triggered in the view thread need to bubble only in the view thread, but should call event listeners in the data thread. Therefore, data thread needs a method to directly call event listeners but not bubble.